framework/src/ble_manager/ble_manager_state.c

```diff
@@ -29,8 +29,12 @@
 #include "ble_manager_state.h"
 #include "ble_manager_log.h"
 
+#define SCAN_INFO_BUFFER_SIZE 100
```

Comment:
how about to add a description why did you set this definition to 100.

Reply:
@pillip8282 
Actually, it is just a heuristic value for getting scanned devices.
This can be customized following the system env.

Reply:
I commented it in the code.